akka-operator/values.yaml

```diff
@@ -38,3 +38,22 @@ resources:
 nodeSelector: {}
 tolerations: []
 affinity: {}
+
+# valid values:
+# - nil (default)
+# - aws 
+# - gcp
+#
+# use `--set` in `helm install` to set from command line
+#
+# if (gcp && imageRepository == blank) {
```

Comment:
this logic needs to be implemented, right?
somewhere like: https://github.com/lightbend/akka-operator-helm/blob/master/akka-operator/templates/040-deployment.yaml#L34

Reply:
yeah .. will do it soon ..

Reply:
done now ..